### PR TITLE
[Docs] Update configuration

### PR DESCRIPTION
**Pull request by Herklos**
Depends on #1487

### CODE REVIEW

File: docs/source/pages/Configuration.rst [hunk 1]
```diff
@@ -36,12 +35,11 @@ Once you have your own **user/config.json** file, to start using OctoBot, you wi
...
        "EXCHANGE_NAME": {
          "api-key": "",
-         "api-secret": "",
-         "web-socket": true
+         "api-secret": ""
        }
    }
 
-`Here are the docs helping to setup an exchange for OctoBot <Exchanges.html>`_
+`Here is the wiki page helping to setup an exchange for OctoBot <https://drakkar-software.github.io/OctoBot/pages/Exchanges.html>`_
```
Comment (GuillaumeDSM) [inline docs/source/pages/Configuration.rst:42]: Here is the **doc** page ?

File: docs/source/pages/Configuration.rst [hunk 2]
```diff
@@ -52,7 +50,7 @@ CryptoCurrencies
    :alt: currencies
 
 
-OctoBot will trade all the cryptocurrencies listed in its configuration. To tell which cryptocurrencies to trade, add the currency in the **crypto-currencies** section in **user/config.json**. 
+OctoBot will trade all the cryptocurrencies listed in its configuration. To tell which cryptocurrencies to trade, add the currency in the **crypto-currencies** section in **user/profiles/YOUR-PROFILE/profile.json**.
```
Comment (GuillaumeDSM) [inline docs/source/pages/Configuration.rst:53]: 👍

File: docs/source/pages/Configuration.rst [hunk 3]
```diff
@@ -111,26 +109,24 @@ Interfaces
...
+  `Here are the details on how to setup a trader simulator. <https://drakkar-software.github.io/OctoBot/pages/Simulator.html>`_
 
 
 .. image:: https://raw.githubusercontent.com/Drakkar-Software/OctoBot/assets/wiki_resources/trading.jpg
    :target: https://raw.githubusercontent.com/Drakkar-Software/OctoBot/assets/wiki_resources/trading.jpg
    :alt: trading
 
 
-Any type of trading has its **risk** parameter. It is a parameter defining the behavior of the trader, similarly to a real human trader. `This risk parameter is described here <Trader.html#risk>`_
+Any type of trading has its **risk** parameter. It is a parameter defining the behavior of the trader, similarly to a real human trader. `This **risk** parameter is described here <https://drakkar-software.github.io/OctoBot/pages/Trader.html#risk>`_
```
Comment (GuillaumeDSM) [inline docs/source/pages/Configuration.rst:129]: is it not working on your setup ? it works on mine and on deployed pages. I think we should avoid using the full URL to be able to change it if necessary without having to update each and every link

Comment (Herklos) [reply docs/source/pages/Configuration.rst:129]: We should also avoid using hardcoded html file name

**Pull request closed without merging**